data/181/380/923/3/1813809233.geojson

```diff
@@ -51,8 +51,8 @@
         }
     ],
     "wof:id":1813809233,
-    "wof:lastmodified":1683053936,
-    "wof:name":"Umrilalman",
+    "wof:lastmodified":1702578908,
+    "wof:name":"Urjagram Tadali",
```

Comment:
Should the English preferred name also update?

Reply:
Ah, yes! I made this change, but didn't push that commit.. pushing now.